modules/ROOT/pages/con_pipeline-image-resources.adoc

```diff
@@ -0,0 +1,91 @@
+[]
+= Image Resource
+
+An Image resource represents an image that lives in a remote repository. It is
```

Comment:
@joaedwar: Do we need "that lives" here? "An Image resource represents an image in a remote repository" sounds good enough to me. What do you think?